### PR TITLE
Update documentation to introduce `Subscriptions`

### PR DESCRIPTION
**Pull request by pfferrari**
## What I did

- Updated readme to introduce `Subscriptions` brand new section of the MFE app
- Replaced old images to show the updated menu

## Checklist

- [x] Make sure your changes are tested (stories and/or unit, integration, or end-to-end tests).
- [x] Make sure to add/update documentation regarding your changes.
- [x] You are **NOT** deprecating/removing a feature.

### CODE REVIEW

File: packages/my-account/README.md [hunk 1]
```diff
@@ -71,38 +71,71 @@ This works as the My Account app's welcome page, showing a list of the customer'
...
+
+- An introductory recap about subscription settings (like start date, frequency, next run date and address)
+- The subscription summary information with the list of subscription line items.
+- The payment method used for the subscription renewal.
+- The history of subscription's related orders.
+  
+IMAGE
+
+It could happen that for any kind of reason the subscription is no longer able to be renewed, for example because of the expiry of the associated payment method.
+In this case, an alert is shown at the top of detail page to highlight the issue and provide the customer with a valid checkout link to place the order with a valid payment method and let the subscription payment method be updated for future runs.
```
Comment (malessani) [inline packages/my-account/README.md:126]:
```suggestion
This alert highlights the issue and provides a link for the customer to checkout the order with a valid payment method, thereby updating the payment information for future transactions.
```

File: packages/my-account/README.md [hunk 2]
```diff
@@ -71,38 +71,71 @@ This works as the My Account app's welcome page, showing a list of the customer'
...
+
+### Subscriptions
+
+#### Subscriptions history
+
+By clicking on the _Subscriptions_ menu item you can see the history of your subscriptions (if any).
+
+IMAGE
+
+The list is paginated and can be sorted based on each column piece of information:
```
[the remark this answers is not in the document]
Comment (pfferrari) [reply packages/my-account/README.md:107]: If we are ok with this change the same paragraph related to the orders needs to be aligned as well.

File: packages/my-account/README.md [hunk 3]
```diff
@@ -71,38 +71,71 @@ This works as the My Account app's welcome page, showing a list of the customer'
...
+By clicking on one of the subscription numbers in the history list you can enter the selected subscription's details page where you can see:
+
+- An introductory recap about subscription settings (like start date, frequency, next run date and address)
+- The subscription summary information with the list of subscription line items.
+- The payment method used for the subscription renewal.
+- The history of subscription's related orders.
+  
+IMAGE
+
+It could happen that for any kind of reason the subscription is no longer able to be renewed, for example because of the expiry of the associated payment method.
```
[the remark this answers is not in the document]
Comment (pfferrari) [reply packages/my-account/README.md]: Ok! Applied!

File: packages/my-account/README.md (hunk 1 above)
Comment (pfferrari) [reply packages/my-account/README.md]: Ok! Applied with a small change suggested by grammar check. I wrote `...provides a checkout link for the customer to place the order...`. Do you think is ok?

File: packages/my-account/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (malessani) [reply packages/my-account/README.md:107]: let's see also what @sfiorucci thinks

[the remark this answers is not in the document]
Comment (sfiorucci) [reply packages/my-account/README.md:107]: not a big deal, both are correct... your call.

**Review by sfiorucci — APPROVED** [review]

**Comment by pfferrari** [comment]
@malessani we completed action needed on our side. do you want to have a final check?

**Pull request merged**